### DISCUSSION TITLE
I use MyVanna Class with ChromaDB and mistral in ollama but can't run tsql which was returned by ollama

### DISCUSSION THREAD

**Discussion by AnchorZhang0817** [vanna-ai/vanna#632] [Q&A]
**Describe the bug**

I built Vanna on my company's desktop successfully, and it can return tsql code by ollama, but it can't run tsql code with throughout 'Couldn't run sql:  (pyodbc.ProgrammingError) ('42S22', "[42S22] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Invalid column name 'Activity'. (207) (SQLExecDirectW)")
(Background on this error at: https://sqlalche.me/e/20/f405)'

**To Reproduce**
1. My Vanna Class:class MyVanna(ChromaDB_VectorStore, Ollama):
    def __init__(self, config=None):
        ChromaDB_VectorStore.__init__(self, config=config)
        Ollama.__init__(self, config=config)

vn = MyVanna(config={'model': 'mistral'})

vn.connect_to_mssql(odbc_conn_str=(....)
2. ask code:vn.ask('How to know one workitem  pass or fail on specific activity?')
3.i had train vanna before
**Expected behavior**

**Error logs/Screenshots**
Couldn't run sql:  (pyodbc.ProgrammingError) ('42S22', "[42S22] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Invalid column name 'Activity'. (207) (SQLExecDirectW)")
[SQL: SELECT..........
]
(Background on this error at: https://sqlalche.me/e/20/f405)

**Desktop (please complete the following information where):**
 - OS: [windows11]
 - Version: [ollama:0.3.0]
 - Python: [3.9.10]
 - Vanna: [2.8.0]

**Additional context**

**Comment by AnchorZhang0817** [comment]
i use real database in ssms to train the Vanna

**Comment by AbdoDabbas** [comment]
What happens if you run the flaskapp ?
Also can you provide the response from ollama ? 

I had this issue where it was generating wrong SQL query that really can't be executed, but this happened when I was using the model `llama3.1` when I used `mistral` it worked, it needed many corrections because it was reading from the wrong tables, but at least it worked.
